docker/docker-compose.yml

```diff
@@ -0,0 +1,127 @@
+version: '3'
+services:
+  # ----- glowing-bear -----
+  glowing-bear:
+    container_name: glowing-bear
+    image: 'thehyve/glowing-bear'
+    ports:
+      - 80:80
```

Comment:
Maybe good to expose this at a port > 1024, because otherwise this only works for root users and if you don't already have a webserver running.